test/lit/branch-hinting.wast

```diff
@@ -259,6 +259,128 @@
     )
   )
 
+  ;; CHECK:      (func $branch-hints-if-else (type $0) (param $x i32)
```

Comment:
In addition to this test, can we have another test that directly shows the correct binary offsets? Or do we not have a way to print IR annotated with binary offsets?

Reply:
Hmm, we don't have an option to show binary offsets by a flag. We show them when DWARF sections exist, or code annotations, atm. Might be worth adding though.